### DISCUSSION TITLE
What are the best features for a CMS

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#6882] [Ideas]
What kind of features do you expect when you choose a CMS?

What features do you like the most in other CMSs you already used?
Examples: 
- Guttenberg layout editor in Wordpress.
- Drag and drop editor in CraftCMS.
- Liquid templates in Shopify.
- Views in Drupal.

**Comment by iafilm-makers** [comment]
Blog  
  
Does "blog" exist in OrchardCore CMS? I cannot find it in the Admin area. The documentation github link redirects to classic Orchard CMS.

![Capture_Orchard-Admin-Menu](https://user-images.githubusercontent.com/4345497/90327080-15d46500-dfe4-11ea-96a6-72cb1d43eb41.JPG)
![Capture_Orchard-Blog-Docu](https://user-images.githubusercontent.com/4345497/90327081-179e2880-dfe4-11ea-92b9-bcd69f1de75f.JPG)

**Reply by Piedone** [reply]
It exists but it's not a feature on its own but rather a specific configuration of other features. Meaning, you can throw together a blog with what's built into Orchard. If you set up your site with the Blog recipe then that's exactly what happens.

**Comment by agriffard** [comment]
> The documentation github link redirects to classic Orchard CMS.

What link you are talking about and where is it exactly?

**Reply by hishamco** [reply]
Even I don't know which one he means? I already checked both the README.md & Wiki everything LGTM

**Reply by iafilm-makers** [reply]
[https://docs.orchardcore.net/projects/O1/en/latest/Documentation/Adding-a-Blog-to-Your-Site/](https://docs.orchardcore.net/projects/O1/en/latest/Documentation/Adding-a-Blog-to-Your-Site/)  
  
I do not remember exactly how my searching for orchardcore blog information led me to the above.  In trying to repeat my navigation tonight I find this more helpful info:
  
[https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms/#creating-a-blog-post-content-type](https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms/#creating-a-blog-post-content-type)

**Comment by dodyg** [comment]
What I miss in OrchardCore is the ability to edit CSS/LESS/SASS/JS files directly online with decent editor support. The theme is nice but it is really restrictive. You have to redeploy to make any changes to the site.
